### PR TITLE
Add Service Examples

### PR DESCRIPTION
**Pull request by ronenhamias**
resolve conflicts and update with latest develop

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -96,6 +96,7 @@
     <module>services-codec-jackson</module>
     <module>services</module>
     <module>benchmarks</module>
+    <module>scalecube-services-examples</module>
```
Comment (segabriel) [inline pom.xml]: maybe rename to `examples` or `services-examples`.

Comment (artem-v) [reply pom.xml]: Use just `examples` . We will all examples there. Not only service related.

File: scalecube-services-examples/pom.xml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+			<artifactId>scalecube-services</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>it.unimi.dsi</groupId>
+			<artifactId>fastutil</artifactId>
+			<version>8.1.1</version>
+		</dependency>
+	</dependencies>
+</project>
```
Comment (artem-v) [inline scalecube-services-examples/pom.xml]: Add a new line

File: scalecube-services-examples/pom.xml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```
Comment (artem-v) [inline scalecube-services-examples/pom.xml]: Format xml with our code formatter

**Review by artem-v — APPROVED** [review]

**Pull request merged**